tests/integration/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Do you think it makes sense to verify that the volume/network are actually setup as expected?

Networks are exposed through the metadata service. I do not think that the volume name is available to the server, but we could at least check that there is at least 1 block devices at `/dev/disk/by-id/scsi-0HC_Volume_*` available?

Reply:
We can add a check to ensure everything is working. That said, I don't think it is the responsibility of the molecule driver to ensure this is working, so I would not put too much effort into this.

Reply:
No, we definitely dont need to verify that it _actually_ works, we should assume that.

Right now we have no checks that anything related to volumes or networks happens at all, thats why I suggested adding some kind of test, and this was the easiest. We could of course also check that volume & network are attached by using the `_info` modules.